languages.toml

```diff
@@ -1316,6 +1316,16 @@ injection-regex = "yml|yaml"
 name = "yaml"
 source = { git = "https://github.com/ikatyang/tree-sitter-yaml", rev = "0e36bed171768908f331ff7dff9d956bae016efb" }
 
+[[language]]
+name = "nestedtext"
+scope = "text.nested"
+injection-regex = "nestedtext"
+file-types = ["nt"]
+roots = []
+comment-token = "#"
+indent = { tab-width = 4, unit = "    " }
+grammar = "yaml"
```

Comment:
The `roots` and `grammar` field can be eliminated since they're using the default values

Reply:
Roots is OK, but I don't think removing `grammar` works.
At least in my setup after I remove it and rebuild I don't get any syntax highlighting and autoindent.

Reply:
Oh whoops, you're right, `yaml` wouldn't be the default value here

Reply:
Done